newdle/client/src/components/Navigator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Container} from 'semantic-ui-react';
+import {Step, StepBody} from './Step';
```

Comment:
Just wondering.. do we need a separate file for this component? It's somewhat small and only useful within the context of `Navigator`, so I'd rather define it in the same file (and not export it)